engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/FetchCachedResultsActor.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.workflow.lifecycle.execution.callcaching
+
+import akka.actor.{Actor, ActorLogging, Props}
+import cromwell.Simpletons._
+import cromwell.core.{DoesNotExpectMessages, JobOutputs}
+import cromwell.core.simpleton.WdlValueBuilder
+import cromwell.database.CromwellDatabase
+import cromwell.database.sql.MetaInfoId
+import cromwell.engine.workflow.lifecycle.execution.callcaching.FetchCachedResultsActor.{CachedOutputLookupFailed, CachedOutputLookupSucceeded}
+import wdl4s.TaskOutput
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+object FetchCachedResultsActor {
+  def props(metaInfoId: MetaInfoId, taskOutputs: Seq[TaskOutput]): Props =
+    Props(new FetchCachedResultsActor(metaInfoId, taskOutputs))
+
+  sealed trait CachedResultResponse
+  case class CachedOutputLookupFailed(metaInfoId: MetaInfoId, failure: Throwable) extends CachedResultResponse
```

Comment:
I'm going to continue to tilt my anti-Throwable windmill, but as @cjllanwarne it's pretty endemic already so meh